modules/oe_migration_workflow/tests/src/Unit/Plugin/migrate/process/SetWorkflowStateTest.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_migration_workflow\Unit\Plugin\migrate\process;
+
+use Drupal\Core\Config\ConfigManagerInterface;
+use Drupal\Core\Config\Entity\ConfigEntityTypeInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState;
+use Drupal\migrate\Row;
+use Drupal\Tests\migrate\Unit\process\MigrateProcessTestCase;
+use Drupal\workflows\WorkflowInterface;
+use stdClass;
+
+/**
+ * @coversDefaultClass \Drupal\oe_migration_workflow\Plugin\migrate\process\SetWorkflowState
+ *
+ * @group oe_migration
+ */
+class SetWorkflowStateTest extends MigrateProcessTestCase {
+  /**
+   * The ID of the plugin under test.
+   *
+   * @var string
+   */
+  protected $pluginId = 'oe_migration_workflow_set_workflow_state';
+
+  /**
+   * The instance of the plugin under test.
+   *
+   * @var \Drupal\migrate\ProcessPluginBase
+   */
+  protected $plugin;
+
+  /**
+   * Destination property, only to show errors.
+   *
+   * @var string
+   */
+  protected $destinationProperty = 'Destination test property';
+
+  /**
+   * The EntityTypeManager mock object.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The ConfigManager mock object.
+   *
+   * @var \Drupal\Core\Config\ConfigManagerInterface
+   */
+  protected $configManager;
+
+  /**
+   * Process plugin configuration.
+   *
+   * @var array
+   */
+  protected $configuration;
```

Comment:
Unsure if you need this one here either? Better to just pass it to initializePlugin no? More readable

Reply:
mm... maybe, yes. Changed.

Reply:
@samuelsolis Yes, but you left there the variable.

Reply:
:facepalm: fixed